maps/donut2.dmm

```diff
@@ -17922,6 +17922,7 @@
 /obj/cable{
 	icon_state = "4-8"
 	},
+/obj/mapping_helper/access/security,
```

Comment:
why sec and not hos?

Reply:
Donut 2's back door is sec access, and the front one is HoP. So more QOL seeing that this one is technically "half solved" in regards to this PR

Reply:
It's actually on purpose not sec access for the front door so a hop can escape if sec comes in from the back to kick their teeth in for handing out AA

Reply:
gotcha, reverting